app/assets/components/elements/json-delete.html

```diff
@@ -1,9 +1,18 @@
-<polymer-element name=json-delete extends=json-req>
-
-    <script>
-        Polymer({
-            method: 'delete'
-        });
-    </script>
-
-</polymer-element>
+<!--
+    TODO(polyup): Inheriting from other custom elements is not yet supported.
+    See: https://www.polymer-project.org/1.0/docs/migration.html#inheritance
```

Comment:
Yeah. Its a fairly big change. Happy to work on it but we should figure out
a plan of attack.

There is a tool called polyup that can automate many of the changes here :
https://www.polymer-project.org/1.0/docs/migration.html
Also a good guide here
http://jcrowther.io/2015/06/01/upgrading-from-polymer-v0-5-v1-0/

Might be worth a quick call to plan how we are going to do this?

Cheers

Stuart

On Thu, Jan 7, 2016 at 9:45 AM Arfon Smith <notifications@github.com> wrote:

> In app/assets/components/elements/json-delete.html
> <https://github.com/openjournals/theoj/pull/256#discussion_r49079222>:
>
> > @@ -1,9 +1,18 @@
> > -<polymer-element name=json-delete extends=json-req>
> > -
> > -    <script>
> > -        Polymer({
> > -            method: 'delete'
> > -        });
> > -    </script>
> > -
> > -</polymer-element>
> > +<!--
> > +    TODO(polyup): Inheriting from other custom elements is not yet supported.
> > +    See: https://www.polymer-project.org/1.0/docs/migration.html#inheritance
>
> @stuartlynn <https://github.com/stuartlynn> @marcrohloff
> <https://github.com/marcrohloff> it looks like we're going to need to
> change some of the element inheritance stuff here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openjournals/theoj/pull/256/files#r49079222>.
>
-- 
*Stuart Lynn*
Map Scientist

*CartoDB—Map Your World's Data*
143 Roebling St 2A, New York, 11211, USA
T: +3123940797  Twitter: @stuart_lynn
W: www.cartodb.com

Live sessions to learn CartoDB